src/web/src/components/expenses/Expense.js

```diff
@@ -30,7 +41,12 @@ const Expense = () => {
   };
   return (
     <div className={classes.container}>
-      <ImgTable />
+      <div className={classes.imgTable}>
+        <ImgTable />
+      </div>
+      <div className={classes.addReceiptButton}>
+        <AddReceiptButton />
+      </div>
```

Comment:
For code resistancy, can you move this button into `ImgTable`, similar to this
https://github.com/TueeNguyen/SplitWise3/blob/5ab398ffb5e8896afedeee0a8bf31354ac5465c6/src/web/src/components/expenses/expenseForms/ReceiptTable.js#L157-L167